admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,40 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. You can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired set of nodes.
+
+For example, if you have a OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods into a specific region. 
+
+To create a project with a node selector, use the `--node-selector` option. For example, the following creates a new project named `MyProject`, but runs only on the nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project MyProject --node-selector='type=user-node,region=east'
+----
+====
```

Comment:
The project name can't have capital letters:

```
The project "MyProject" is invalid.
metadata.name: invalid value 'MyProject', Details: must be a DNS label (at most 63 characters, matching regex [a-z0-9]([-a-z0-9]*[a-z0-9])?): e.g. "my-name"
```

It may be worth adding a [NOTE] box here to clarify that `--node-selector` is only available with the `oadm` version of `new-project` and is therefore only a cluster admin operation. The `oc new-project` command doesn't have the `--node-selector` option, so non-cluster-admins cannot use the `--node-selector` option when self-provisioning projects:

```
$ oc new-project my-project-2 --node-selector='type=user-node,region=east'
Error: unknown flag: --node-selector
```